components/FlashcardList.js

```diff
@@ -16,19 +16,33 @@ export default function FlashcardList({
   handleToggleCorrect,
   handleDeleteFlashcard,
   handleUpdateFlashcard,
+  listMode,
 }) {
-  const unansweredFlashcards = flashcards.filter(
-    (flashcard) => flashcard.isCorrect === false
-  );
+  let FlashcardsForList;
+  let TextForZeroFlashcards;
+
+  switch (listMode) {
+    case "archive":
+      FlashcardsForList = flashcards.filter((flashcard) => flashcard.isCorrect);
+      TextForZeroFlashcards =
+        "You have not yet answered any questions in this collection correctly.";
+      break;
+    default:
+      FlashcardsForList = flashcards.filter(
+        (flashcard) => flashcard.isCorrect === false
+      );
+      TextForZeroFlashcards = "There are no flashcards in this collection.";
+      break;
+  }
```

Comment:
sorry but this code will hurt you in the long run. Instead of doing this, please add two props to your Comopnent for passing down different flashcardsLists for different purposes.
```jsx
function FlashcardList({flashCards,message}) {
 ...
 flashCards.map(...)
}
```
```jsx
// collections/archive.js
...
<FlashcardList flashCards={archivedCards} />
```

```jsx
// collections/[id].js
...
<FlashcardList flashCards={cardsFromSelectedCollection} />
```